client/branded/src/components/panel/TabbedPanelContent.tsx

```diff
@@ -141,10 +142,11 @@ export const TabbedPanelContent = React.memo<TabbedPanelContentProps>(props => {
     const areExtensionsReady = useObservable(
         useMemo(() => haveInitialExtensionsLoaded(props.extensionsController.extHostAPI), [props.extensionsController])
     )
-
+    const [redesignedEnabled] = useTemporarySetting('codeintel.referencePanel.redesignedEnabled', false)
     const isExperimentalReferencePanelEnabled =
-        !isErrorLike(props.settingsCascade.final) &&
-        props.settingsCascade.final?.experimentalFeatures?.coolCodeIntel === true
+        (!isErrorLike(props.settingsCascade.final) &&
+            props.settingsCascade.final?.experimentalFeatures?.coolCodeIntel === true) ||
+        redesignedEnabled
```

Comment:
```suggestion
        redesignedEnabled === true
```

For consistency, we do it this way in BlobPanel.